__tests__/news-server.test.js

```diff
@@ -32,13 +32,48 @@ describe("GET /api/topics", () => {
         });
       });
   });
-
   test("404: responds with an error message when given a path that does not exist", () => {
     return request(app)
       .get("/api/notAPath")
       .expect(404)
       .then(({ body }) => {
-        expect(body.msg).toBe("Error: path does not exist");
+        expect(body.msg).toBe("Error: 404 not found");
+      });
+  });
+});
+
+describe("GET /api/articles", () => {
```

Comment:
this endpoint doesnt match, this might make debugging difficult in the future, I would recommend having "GET /api/articles/:article_id" as the describe block title